extras/encryption/pubspec.yaml

```
@@ -7,8 +7,8 @@ environment:
   sdk: '>=2.12.0-0 <3.0.0'
 
 dependencies:
-  moor: ^4.0.0-nullsafety.0
-  sqflite_sqlcipher: ^2.0.0-nullsafety
+  moor: '>=4.0.0 <5.0.0'
+  sqflite_sqlcipher: '>=2.0.0 <3.0.0'
```

I think those are equivalent to the following which IMO are easier to read

```suggestion
  moor: ^4.0.0
  sqflite_sqlcipher: ^2.0.0
```

---

Ok, sounds good!